sdk/resourcemanager/trustedsigning/armtrustedsigning/certificateprofiles_client_example_test.go

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) Go Code Generator. DO NOT EDIT.
+
+package armtrustedsigning_test
+
+import (
+	"context"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/trustedsigning/armtrustedsigning"
+	"log"
+	"time"
+)
+
+// Generated from example definition: D:/ws/azure-sdk-for-go/sdk/resourcemanager/trustedsigning/armtrustedsigning/TempTypeSpecFiles/CodeSigning.Management/examples/2024-02-05-preview/CertificateProfiles_Create.json
```

Comment:
```suggestion
// Generated from example definition: 2024-02-05-preview/CertificateProfiles_Create.json
```

please do this change manually for all example file